### ISSUE TITLE
Update the binary on Winget

### ISSUE THREAD

**Issue by rehanhooda** [namazso/OpenHashTab#82]
Hi dev,

Could you update the OpenHashTab file on the winget repo? Currently, it is showing 2.3.0 version there.

![image](https://user-images.githubusercontent.com/62488621/123731631-261cc800-d8b6-11eb-859f-ead24e283f6a.png)

Thanks,

**Comment by MisterX2000** [comment]
Yeah, I've also noticed that the pull request here https://github.com/microsoft/winget-pkgs/pull/7037 has already been merged since February, but it's still not showing with the winget-cli.
But I also have no Idea, how the process behind it works, so  ¯\_(ツ)_/¯

**Comment by namazso** [comment]
to be honest.. me neither, i just copied what the pr before me did

**Comment by djdv** [comment]
In addition to the previous update PR, I saw this one https://github.com/microsoft/winget-pkgs/pull/21601
Which targets v3.0.1 but I'm still seeing 
```
> winget search openhashtab
Name        Id                  Version Source
-----------------------------------------------
OpenHashTab namazso.OpenHashTab 2.3.0   winget
```

Maybe an issue on their side that needs to be reported/resolved. It looks like their CI didn't like so maybe it's related.

**Comment by NotaInutilis** [comment]
Digging this issue up a bit because, if i remember correctly, v3.0.1 was showing up at one point in the winget tool.

Also, I'd like to risk a barely educated guess: can this issue be caused by the "v" prefix? It's only on the [3.0.1 release](https://github.com/microsoft/winget-pkgs/blob/master/manifests/n/namazso/OpenHashTab/v3.0.1/namazso.OpenHashTab.installer.yaml), and not on the [2.3.0 one](https://github.com/microsoft/winget-pkgs/blob/master/manifests/n/namazso/OpenHashTab/2.3.0/namazso.OpenHashTab.yaml). Moreover, I believe it would make sense for an upgrade tool to put letters before numbers in its ordering system, because of some release going "alpha x" or "beta y" before 1.0.

**Comment by sitiom** [comment]
> Also, I'd like to risk a barely educated guess: can this issue be caused by the "v" prefix? It's only on the [3.0.1 release](https://github.com/microsoft/winget-pkgs/blob/master/manifests/n/namazso/OpenHashTab/v3.0.1/namazso.OpenHashTab.installer.yaml?rgh-link-date=2021-09-20T13%3A13%3A07Z), and not on the [2.3.0 one](https://github.com/microsoft/winget-pkgs/blob/master/manifests/n/namazso/OpenHashTab/2.3.0/namazso.OpenHashTab.yaml?rgh-link-date=2021-09-20T13%3A13%3A07Z).

You're right. I have created a PR to fix this:
- #188

As for auto-updating the Winget manifest, I have also created a PR to setup the Winget Releaser workflow:
- #189

**Comment by sitiom** [comment]
> This was fixed on the Winget side already:
> 
> ![image](https://user-images.githubusercontent.com/8676443/256175736-5f2828fe-8b0c-4a4c-a572-bd4625a8b97e.png)

That's because the `2.x` versions were removed in Winget: https://github.com/microsoft/winget-pkgs/pull/39284, https://github.com/microsoft/winget-pkgs/pull/39284. I think the better solution was to trim off the `v` prefix on the OpenHashTab side.

**Comment by NotaInutilis** [comment]
There seems to still be some wonky stuff with winget. It didn't upgrade OpenHasTab from 3.0.2 to 3.0.4 automatically with `upgrade --all`, I had to specifically ask for this package to upgrade.
Plus, the `v` prefix does not appear in OpenHashTab's `Check update` window, having a `version x` instead, so it'd better be cut off (or added) everywhere for consistency.
